### ISSUE TITLE
The training epochs of KIBA

### ISSUE THREAD

**Issue by YifanDengWHU** [kexinhuang12345/DeepPurpose#70]
Hi, Kexin. I'm writing to ask about the reproduction of DeepPurpose. Here I want to get the result for MPNN+AAC in the KIBA dataset. However, it seems that 150 epochs aren't enough for KIBA while they work for DAVIS. I can only get 0.73 for C-index, much smaller than that in your paper. So I wonder how many epochs need to be set while training KIBA.

**Comment by kexinhuang12345** [comment]
That's weird. @futianfan do you remember what is the parameter set for MPNN+AAC for KIBA?

**Comment by futianfan** [comment]
Sorry, I don't remember, @YifanDengWHU , did you change the hyperparameter of MPNN?  Or follow the hyperparams in tutorial for MPNN? I think 150 epochs is sufficient.

**Comment by futianfan** [comment]
@YifanDengWHU what is your MSE and how does learning curve looks like

**Comment by YifanDengWHU** [comment]
Hi! @futianfan. I didn't notice the tutorial in the repo at first. Here I follow the hyperparameters in it and perform experiment again and hope it may work.
I used the default hyperparameters at first and I only changed the training epochs to 150, then I found that the loss didn't go down steadily at the first few epochs. So I decreased the LR to 1e-5 and increased the training epochs to 300. It seems that it also work now but the training time is too long.  (It hasn't finished yet so the learning curve can't be seen.)
Let me finish the experiment with the same hyperparameters in the tutorial at first. Thanks you so much for your help!
